src/ORM/FieldType/DBForeignKey.php

```diff
@@ -29,6 +29,15 @@ class DBForeignKey extends DBInt
      */
     protected $object;
 
+    /**
+     * This represents the number of related objects to show in a dropdown before it reverts
+     * to a NumericField
+     *
+     * @config
+     * @var int
+     */
+    private static $dropdownFieldThreshold = 100;
```

Comment:
Config vars should always be `$lower_case_underscore` (except for when they’re not, `many_many_extraFields`)

Reply:
snek_case_like_a_good_boi